MdePkg/Include/Register/Intel/StmApi.h

```diff
@@ -52,10 +52,19 @@ typedef struct {
   ///
 } SOFTWARE_STM_HEADER;
 
+// MU_CHANGE - [START]: Add new CPU information header for STM
 typedef struct {
-  MSEG_HEADER            HwStmHdr;
-  SOFTWARE_STM_HEADER    SwStmHdr;
+  UINT32    NumberOfCpus;

```

Comment:
I think it'd be best to add a unique signature to the beginning of this structure. Then, a consumer could generically test for it at MSEG header + 2k to see if this structure is present.

Reply:
That makes sense.  Updated.